modules/nw-ingress-sharding.adoc

```diff
@@ -13,5 +15,90 @@ As the primary mechanism for traffic to enter the cluster, the demands on the In
 * Allow certain Ingress Controllers to have different policies defined.
 * Allow only specific routes to use additional features.
 * Expose different routes on different addresses so that internal and external users can see different routes, for example.
+* Transfer traffic from one version of an application to another during a blue green deployment.
 
-Ingress Controller can use either route labels or namespace labels as a sharding method.
+When Ingress Controllers are sharded, a given route is bound to zero or more routers in the group. The route binding ensures uniqueness of the route across the shard. Uniqueness allows secure and non-secure versions of the same route to exist within a single shard. This implies that routes now have a visible life cycle that moves from created to bound to active.
+
+Ingress Controller can use three sharding methods:
+
+. Adding only a namespace selector to the Ingress Controller, so that all routes in that namespace are in the Ingress shard.
```

Comment:
Wording seems to suggest the fact labels aren't involved.

```suggestion
. Adding only a namespace selector to the Ingress Controller, so that all routes in a namespace with labels that match the namespace selector are in the Ingress shard.
```